app/(company)/declare/components/DeclarePage.js

```diff
@@ -0,0 +1,132 @@
+'use client';
```

Comment:
this should not be a client component. get the data on the server and pass it here. No state is needed.